docs/development/updating_tor.rst

```diff
@@ -0,0 +1,32 @@
+Updating Tor
+============
+
+Given SecureDrop's significant reliance on Tor via Onion Services, we
+test new Tor versions to ensure they don't break SecureDrop before releasing
+them to users.
+
+Identifying new releases
+------------------------
+
+Announcements for new Tor releases are posted in the `Tor forum
+<https://forum.torproject.net/c/news/tor-release-announcement/28>`_.
+
+Our continuous integration automatically checks for new Tor packages every
+night and should commit them to the `securedrop-dev-packages-lfs
+<https://github.com/freedomofpress/securedrop-dev-packages-lfs>`_ repository.
+Within 15 minutes they should be available for download via
+``apt-test.freedom.press``.
+
+Testing
+-------
+
+Use a staging environment to verify that with the new Tor release, SecureDrop
```

Comment:
Ideally, a prod environment should be used and left to run through the unattended-upgrade cycle, to confirm that Tor will still be on the correct version and running as expected after a reboot.

Reply:
Added, mostly using your wording here :)